Test/TestCases/node-addon-api/object/delete_property.cs

```diff
@@ -23,7 +23,7 @@ private static JSValue DeletePropertyWithLatin1StyleString(JSCallbackArgs args)
     {
         JSValue obj = args[0];
         JSValue key = args[1];
-        return obj.DeleteProperty(JSNativeApi.CreateStringLatin1(key.GetValueStringLatin1()));
+        return obj.DeleteProperty(JSValue.CreateStringLatin1(key.GetValueStringLatin1()));
```

Comment:
Should there be a `JSString` type?

Reply:
Good catch! I will change it.

Reply:
It doesn't need to be done in this PR.

Reply:
We do not have `JSString` yet. I can change it later when we add it.